src/Sylius/Bundle/AttributeBundle/Resources/views/attributeChoice.html.twig

```diff
@@ -0,0 +1,26 @@
+<div class="modal fade" id="attributes-modal" role="dialog">
```

Comment:
All these templates are specific to Sylius UI. I am ok to leave them here right now, but we should definitely simplify the default templates shipped with the bundle itself and override them in Sylius Admin bundle to provide proper view later.